config/routes.rb

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
   root "welcome#index"
+
+  resources :foods do
```

Comment:
No need for a `do end` here. It's best practice to only create the routes you're going to use, so in this case an `resources :foods, only: [:index]` would be ideal.